packages/messaging-api-line/src/LineTypes.ts

```diff
@@ -366,23 +366,188 @@ export type FlexBoxContent =
+   *     container specifies the order.
+   * - `vertical`: Components are placed vertically from top to bottom.
+   * - `baseline`: Components are placed in the same way as `horizontal` is
+   *     specified except the baselines of the components are aligned.
+   *
+   * For more information, see
+   * [Types of box layouts](https://developers.line.biz/en/docs/messaging-api/flex-message-layout/#box-layout-types).
+   */
+  layout: L;
+  /**
+   * Components in this box. Here are the types of components available:
+   *
+   * - When the `layout` property is `horizontal` or `vertical`:
+   *   + [Box](https://developers.line.biz/en/reference/messaging-api/#box)
+   *   + [button](https://developers.line.biz/en/reference/messaging-api/#button)
+   *   + [image](https://developers.line.biz/en/reference/messaging-api/#f-image)
+   *   + [text](https://developers.line.biz/en/reference/messaging-api/#f-text)
+   *   + [separator](https://developers.line.biz/en/reference/messaging-api/#separator)
+   *   + [filler](https://developers.line.biz/en/reference/messaging-api/#filler)
+   *   + [spacer (not recommended)](https://developers.line.biz/en/reference/messaging-api/#spacer)
+   * - When the `layout` property is `baseline`:
+   *   + [icon](https://developers.line.biz/en/reference/messaging-api/#icon)
+   *   + [text](https://developers.line.biz/en/reference/messaging-api/#f-text)
+   *   + [filler](https://developers.line.biz/en/reference/messaging-api/#filler)
+   *   + [spacer (not recommended)](https://developers.line.biz/en/reference/messaging-api/#spacer)
+   */
+  contents: L extends 'baseline'
+    ? (FlexIcon | FlexText | FlexFiller | FlexSpacer)[]
+    : (
+        | FlexBox<FlexBoxLayout>
+        | FlexButton
+        | FlexImage
+        | FlexText
+        | FlexSeparator
+        | FlexFiller
+        | FlexSpacer
+      )[];
+  /**
+   * Background color of the block. In addition to the RGB color, an alpha
+   * channel (transparency) can also be set. Use a hexadecimal color code.
+   * (Example:#RRGGBBAA) The default value is `#00000000`.
+   */
+  backgroundColor?: string;
+  /**
+   * Color of box border. Use a hexadecimal color code.
+   */
+  borderColor?: string;
+  /**
+   * Width of box border. You can specify a value in pixels or any one of none,
+   * light, normal, medium, semi-bold, or bold. none does not render a border
+   * while the others become wider in the order of listing.
+   */
+  borderWidth?: 'none' | 'light' | 'normal' | 'medium' | 'semi-bold' | 'bold';
+  /**
+   * Radius at the time of rounding the corners of the border. You can specify a
+   * value in pixels or any one of `none`, `xs`, `sm`, `md`, `lg`, `xl`, or `xxl`. none does not
+   * round the corner while the others increase in radius in the order of listing. The default value is none.
+   */
+  cornerRadius?: 'none' | 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl';
```

Comment:
cornerRadius?: Size

But there are many type of size combinations. it's hard to naming each type of size.

Reply:
See PR: https://github.com/Yoctol/messaging-apis/pull/561

Reply:
Removed all `Size`s